corehq/apps/userreports/reports/view.py

```diff
@@ -104,7 +104,7 @@ def query_dict_to_dict(query_dict, domain):
     :param domain:
     :return: a dict
     """
-    request_dict = json_request(query_dict)
+    request_dict = json_request(query_dict, booleans_as_strings=True)
```

Comment:
I think yes.
Booleans in UCR are modeled as `0` or `1`, so filters actually need to be of type `integer`. There is no `boolean` filter type.